tests/unit/test_merlion_handler.py

```diff
@@ -0,0 +1,202 @@
+import os
+import time
+from unittest.mock import patch
+import pandas as pd
+
+from mindsdb_sql import parse_sql
+
+# How to run:
+#  env PYTHONPATH=./ pytest tests/unit/test_merlion_handler.py
+# Warning: uncomment and run too many sql-testcase in detector and forecaster will cost a long time
+from mindsdb.integrations.handlers.merlion_handler.adapters import DefaultForecasterAdapter, SarimaForecasterAdapter, \
+    ProphetForecasterAdapter, MSESForecasterAdapter, IsolationForestDetectorAdapter, \
+    WindStatsDetectorAdapter, ProphetDetectorAdapter
+# from mindsdb.integrations.handlers.merlion_handler.adapters import DefaultDetectorAdapter
+from .executor_test_base import BaseExecutorTest
+
+
+class TestMerlion(BaseExecutorTest):
+    def get_m4_abs_path(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        return os.path.join(dir_path, "m4_test_merlion.csv")
```

Comment:
Should we keep the files somwhere else and reference them from there?

Reply:
All the local files have been replaced with github url.

Reply:
Thanks